autovalet_parking/src/autovalet_parking/Parker.py

```diff
@@ -205,7 +201,7 @@ def tagPoseRANSAC(self):
             vote = 0
             for j in range(len(quatArray)):
                 diff = np.abs(1 - np.dot(quatArray[i], quatArray[j]) ** 2)
-                vote = vote + 1 if diff < 1e-5 else vote
+                vote = vote + 1 if diff < 1e-3 else vote
```

Comment:
lol python is weird, didn't know you could do if statements like this

Reply:
I use this a lot :joy: make the code clean

Reply:
similar to ? in c++